src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -75,7 +75,9 @@ const MainNavDroplist = Droplist.extend({
         return currentOrg.get('name');
       },
       infoText() {
-        return 'https://help.roundingwell.com/';
+        const helpUrl = Radio.request('bootstrap', 'setting', 'help_url');
+
+        return helpUrl ?? 'https://help.roundingwell.com/';
```

Comment:
This should automatically accommodate any settings set specifically to a workspace, right?

If a customer wanted different help links per workspace.